frontend/src/helpers/iframeToYoroiMessaging.js

```diff
@@ -26,6 +26,12 @@ export const YoroiCallback = (selectedPools: SelectedPools) => {
 
   return useCallback(() => {
     const encodedDataForYoroi = relevantDataForYoroi(selectedPools)
+    let postMessage = window.parent.postMessage
+    if (window.ReactNativeWebView) {
+      postMessage = function(data) {
+        window.ReactNativeWebView.postMessage(data)
+      }
+    }
```

Comment:
If this code is only used for the case `Source.MOBILE` I would keep it there.

Question: For what case for Source.Mobile `window.ReactNativeWebView` is not true? is this someone like iOS vs Android?

Reply:
The `window.ReactNativeWebView` is injected by `WebView` in react nstive., So it should only exist in yoroi mobile.

I used a proxy variable in case we want to take into account a situation in which seiza is rendered in mobile screens but not exactly in react native.

Reply:
@nicarq I did what you suggested.

And answering your question more clearly: the `window.ReactNativeWebView` should exist for both iOS and Android. But I'm not sure about how `source` is determined, maybe it can be that `source=Source.MOBILE` for any small screen size, so in some cases `window.ReactNativeWebView` can be `null`, I guess.

Reply:
@v-almonacid agree that it should exist on iOS and Android, but `window.ReactNativeWebView` shouldn’t be non-null if it’s not using a webview afaik, If you are loading the page from mobile. 

The only way that it should be using a webview it’s a crappy mobile browser is trying to render Seiza.

Reply:
@nicarq `window.ReactNativeWebView` will be null for any previous version of yoroi mobile.

I'll need to test older versions too.